src/Pact/Eval.hs

```diff
@@ -164,6 +161,17 @@ enforceGuard i g = case g of
       MDInterface{} -> evalError' i $ "ModuleGuard not allowed on interface: " <> pretty mg
   GUser UserGuard{..} ->
     void $ runSysOnly $ evalByName _ugFun _ugArgs (getInfo i)
+  GCapability CapabilityGuard{..} -> do
+    maybe (return ()) (enforcePactId True) _cgPactId
```

Comment:
This can be replaced by:
```
forM_ _cgPactId $ \pid ->
  enforcePactId True pid
```
I find this Foldable idiom more readable once the reader is used to it. Naming of the point optional, of course.